dchima/goose-backend
# Typescript configuration setup

## What it does
This PR sets up the boilerplate for typescript in node. It also configures a custom path for `./src` throughout the code base. 

## How to test
You can test the linting by runing `nom run lint` or you can run the development server with `npm run start:dev` to make sure the app works as intended and listens in port **3000**. Other than that there's not mush testing to be done here.

## What's next
Either I setup docker for typescript or I setup sequelize. Not sure which to do first. I'll have a think and probably do the easier one.
